src/ModelInit.jl

```diff
@@ -468,6 +472,15 @@ function get_relevant_sources{NumType <: Number}(
   relevant_sources
 end
 
```

Comment:
I know this function is simple, but please add function documentation saying at least what the input and output is.